documentation/doc-Migration_Toolkit_for_Virtualization/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -136,14 +140,15 @@ $ cat << EOF | oc apply -f -
 apiVersion: virt.konveyor.io/v1alpha1
 kind: Migration
 metadata:
-  name: migration-001
+  name: migration_name
```

Comment:
Maybe here also use same as for the plan CR. <migration_name>, so it can be understandable that it's a free text name.